examples/ssx-test-wagmi-react-dapp/package.json

```diff
@@ -25,4 +25,4 @@
     "eslint-config-next": "^12.1.6",
     "typescript": "^4.7.2"
   }
-}
\ No newline at end of file
+}
```

Comment:
Update the project name to `ssx-test-wagmi-react-dapp`

Reply:
Done